pinot-server/src/test/java/org/apache/pinot/server/api/TableSizeResourceTest.java

```diff
@@ -37,9 +37,9 @@ public void testTableSizeNotFound() {
   @Test
   public void testTableSizeDetailed() {
     TableSizeInfo tableSizeInfo = _webTarget.path(TABLE_SIZE_PATH).request().get(TableSizeInfo.class);
-    ImmutableSegment defaultSegment = _indexSegments.get(0);
+    ImmutableSegment defaultSegment = _realtimeIndexSegments.get(0);
```

Comment:
getTableSize() method is often used for offline table. Can you add the test for offline table as well?

Reply:
done.